### PR TITLE
Remove the taskcluster folder

### PR DESCRIPTION
**Pull request by rehandalal**
/cc @escapewindow to make sure i didn't miss anything.

r?

### CODE REVIEW

**Comment by escapewindow** [comment]
@rehandalal you'll need to re-copy over https://github.com/mozilla-extensions/xpi-template/blob/master/.taskcluster.yml to support cloning xpi-template for the taskcluster source, and populate [xpiSigningType](https://github.com/mozilla-extensions/xpi-template/blob/master/.taskcluster.yml#L19) to enable dep-signing on push/PR.

**Review by escapewindow — CHANGES_REQUESTED** [review]
https://github.com/mozilla-extensions/normandy-devtools/pull/153#issuecomment-622606651

**Review by escapewindow — APPROVED** [review]

File: .taskcluster.yml [hunk 1]
```diff
@@ -7,8 +7,16 @@ tasks:
...
+              revision: c78d65237c6ed064b9cbb3cd063b22e9e052a06a
+          template:
+              repo: https://github.com/mozilla-extensions/xpi-template
+              branch: master
           trustDomain: xpi
+          # XXX for private repos, uncomment this line, and comment out the next
+          # githubCloneSecret: project/xpi/xpi-github-clone-ssh
+          githubCloneSecret: ""
+          # XXX use "privileged" or "system" to enable signing
+          xpiSigningType: "privileged"
```
Comment (mythmon) [inline .taskcluster.yml:16]: We can remove the XXX comments here.

**Review by mythmon — APPROVED** [review]

**Comment by rehandalal** [comment]
bors r=@mythmon

**Pull request merged**